.travis.yml

```diff
@@ -36,7 +36,7 @@ before_install:
   - export APPLITOOLS_BATCH_ID=`echo ${TRAVIS_PULL_REQUEST_SHA:=$TRAVIS_COMMIT}`
   - export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY
   - export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_KEY
-  - npm config set '//registry.npmjs.org/:_authToken' ${NPM_TOKEN}
+  - npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN
```

Comment:
let's hope this work, otherwise we're not in a good position
locally having two tokens broke authentication...